utils/chunk-tests.js

```diff
@@ -9,6 +9,7 @@ const runnersMap = new Map([
       min: 1,
       max: 1,
       runners: ['ubuntu-latest', 'macos-latest', 'windows-latest'],
+      nodeVersion: '18',
```

Comment:
Note that Remix requires Node 18 to build, so updating the Unit tests to run using Node 18. I think considering Node 16 is EOL, this is acceptable.

Reply:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** Can we pull this out similar to `test-next-local`?‏

I'd like to ensure the rest of `test-unit` still works under Node 16 while we support it.

Reply:
I switched the test to use an older version of Next.js (v12) instead. This works on Node 16, so the unit tests' node version is no longer being modified.